pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -143,6 +156,10 @@ type processedPods struct {
 func (dswp *desiredStateOfWorldPopulator) Run(sourcesReady config.SourcesReady, stopCh <-chan struct{}) {
 	// Wait for the completion of a loop that started after sources are all ready, then set hasAddedPods accordingly
 	klog.InfoS("Desired state populator starts to run")
+	if !clientgocache.WaitForCacheSync(stopCh, dswp.pvcHasSynced, dswp.pvHasSynced) {
+		klog.Fatal("Unable to sync cache for pvc and pv")
+	}
+	klog.InfoS("Succeed to sync cache for pvc and pv")
```

Comment:
imo, we dont need to log this , even if we log, something like `synced cache...` would do.

Reply:
removed, thanks!

Reply:
on further look, being it in `InfoS`, it should have been fine.. :) ... any way, lets wait for other's review